frontend/public/components/utils/headings.tsx

```diff
@@ -196,25 +199,32 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
               <span className="co-m-pane__heading-badge">{badge}</span>
             )}
             {link && <div className="co-m-pane__heading-link">{link}</div>}
-            {showActions && (
-              <div className="co-actions" data-test-id="details-actions">
-                {hasButtonActions && (
-                  <ActionButtons actionButtons={buttonActions.map((a) => a(kindObj, data))} />
-                )}
-                {hasMenuActions && (
-                  <ActionsMenu
-                    actions={
-                      _.isFunction(menuActions)
-                        ? menuActions(kindObj, data, extraResources, customData)
-                        : menuActions.map((a) => a(kindObj, data, extraResources, customData))
-                    }
-                  />
-                )}
-                {_.isFunction(customActionMenu)
-                  ? customActionMenu(kindObj, data)
-                  : customActionMenu}
-              </div>
-            )}
+            <div className="co-actions" data-test-id="details-actions">
```

Comment:
Only output `div.co-actions` if is needed(preserve existing behavior).
```suggestion
           {(perspective === 'admin' || showActions) && (
             <div className="co-actions" data-test-id="details-actions">
```